modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -34,7 +41,15 @@ class {{classname}}(object):
         }
 
 {{#vars}}
-        self._{{name}} = {{name}}
+        self._{{name}} = None
+{{/vars}}
+
+        # TODO: let required properties as mandatory parameter in the constructor.
+        #       - to check if required property is not None (e.g. by calling setter)
+        #       - ApiClient.__deserialize_model has to be adapted as well
```

Comment:
@easz Thanks for the "TODO" note. 

Will we better off replacing ApiClient.__deserialize_model with [kim](https://github.com/mikeywaites/kim) or similar Python modules?